.editorconfig

```diff
@@ -1,14 +0,0 @@
-# editorconfig.org
```

Comment:
Why was this deleted? IMO this is an important file -- a good example is that now all of the files mark no new line as a change, which makes it a bit harder to review.

Reply:
will check that. When I started from scratch I decided not to implement anything I don't fully know and left those to the end. Never revisited